contracts/reverseRegistrar/ISignatureReverseResolver.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity >=0.8.4;
+
+interface ISignatureReverseResolver {
+    event VersionChanged(bytes32 indexed node, uint64 newVersion);
+    event ReverseClaimed(address indexed addr, bytes32 indexed node);
```

Comment:
This event should probably be part of `IL2ReverseRegistrar`.

Reply:
`ReverseClaimed` is also emitted at `SignatureReverseResolver.setNameForAddrWithSignature` so can't move to `IL1ReverseRegistrar`.